fundamentals/middleware/problem-details-service/Program.cs

```diff
@@ -0,0 +1,85 @@
+using Microsoft.AspNetCore.Http.Features;
+
+var builder = WebApplication.CreateBuilder(args);
+
+// Add services to the container.
+// Learn more about configuring Swagger/OpenAPI at https://aka.ms/aspnetcore/swashbuckle
+builder.Services.AddEndpointsApiExplorer();
+builder.Services.AddSwaggerGen();
```

Comment:
Maybe remove it to make the sample simpler?

Reply:
okay let me do thaat

Reply:
although I think it would create an easier interface for them to test their post request using the swagger UI.
What do you think?